flutter/android/app/src/main/kotlin/com/carriez/flutter_hbb/InputService.kt

```diff
@@ -252,9 +261,88 @@ class InputService : AccessibilityService() {
         }
     }
 
+    @RequiresApi(Build.VERSION_CODES.N)
+    fun onKeyEvent(data: ByteArray) {
+        val keyEvent = KeyEvent.parseFrom(data);
+        val handler = Handler(Looper.getMainLooper())
```

Comment:
can not parse it in rust?

Reply:
It's serialized on rust side by purpose, because I want to use generated KeyEvent class on Kotlin side instead of pass the  fields of it on rust side through ffi that would a lot of parameters and less convenient.

Reply:
Agree